IO/GeoFileParser.cpp

```diff
@@ -306,25 +311,25 @@ bool GeoFileParser::LoadGeometry(Building* building)
                p /= transFilename;
                transFilename = p.string();
 
-               Log->Write("INFO:\tParsing transition from file <%s>", transFilename.c_str());
+               Logging::Info(fmt::format("Parsing transition from file <{:s}>", transFilename));
```

Comment:
why fmt::format is used here?

Reply:
To insert the transFilename into the format string. Before this was done with sprintf